srcpkgs/jamulus/template

```diff
@@ -0,0 +1,20 @@
+# Template file for 'jamulus'
+pkgname=jamulus
+version=3.6.2
+revision=1
+wrksrc=Jamulus${version}
+build_style=qmake
+configure_args="Jamulus.pro"
+hostmakedepends="pkg-config"
+makedepends="qt5-declarative-devel jack-devel"
+depends="jack"
+short_desc="Play music online. With friends. For free"
+maintainer="KawaiiAmber <japaneselearning101@gmail.com>"
+license="GPL-2.0-or-later"
+homepage="https://jamulus.io/"
+distfiles="${SOURCEFORGE_SITE}/project/llcon/Jamulus/${version}/Jamulus-${version}.tar.gz"
```

Comment:
Why sourceforge and not their github repo? It's recommended in https://jamulus.io/wiki/Installation-for-Linux

Reply:
Weird. When I went to get the distfiles, I just went to jamulus.io and clicked "download now," which led to their sourceforge. I didn't even notice that they recommend the github in their guide. I'll change it to use github when I get back home.

Reply:
Updated to use github distfiles.